torch_xla/experimental/xla_sharded_tensor.py

```diff
@@ -102,6 +102,19 @@ def __new__(cls, elem: torch.Tensor, *args, **kwargs):
     r.global_tensor = elem.detach() if r.requires_grad else elem
     return r
 
+  @staticmethod
+  def from_cpu_shards(shards: List[torch.Tensor],
+                      sharding: torch_xla._XLAC.OpSharding,
+                      global_shape: torch.Size = None):
```

Comment:
What's the benefit of providing a global_shape?

Reply:
If the shards are padded, the global_shape can be used to ignore padding.